deploy/operator-role-kubernetes.yaml

```diff
@@ -20,6 +20,9 @@ rules:
   - persistentvolumeclaims
   - configmaps
   - secrets
+  - serviceaccounts
+  - roles
+  - rolebindings
```

Comment:
I'm not sure how this works w.r.t. privilege escalation prevention [1]. My understanding is that the operator service account is not in the same scope, that is same namespace, as the roles and role bindings that are to be created. As, `escalate` nor `bind` verbs are granted, I wonder what am I missing 🤔.

[1] https://kubernetes.io/docs/reference/access-authn-authz/rbac/#privilege-escalation-prevention-and-bootstrapping

Reply:
Yeah, right.. let me explain.. I've added sas, roles and rbs here mostly thinking to future usage, but they are not needed actually. The only thing needed here is view permission for sas because the code checks its existence (but probably can be removed.. let me clean this up a bit).

To install the operator clusterwide, instead, you need to provide those 3 permissions at cluster level and we'll do it in the olm csv.

Reply:
Perfect!